doc/configuration/useridresolvers.rst

```diff
@@ -355,50 +355,32 @@ The available attributes for the ``Attribute mapping`` are:
  * mobile,
  * email.
 
+.. _httpresolver:
+
 HTTP resolver
 .............
 
-.. index:: HTTP resolver
-
-As a summary, This resolver aims to solve user resolution from a custom HTTP API implementation. For example, you could
-have a custom API at `http://domain.com/my/path` and you want to retrieve information from there. Below, we describe how this
-resolver should be configured.
+.. index:: HTTP resolver, resolver, api, http
 
-The HTTP Resolver contains a few fields to be completed, as shown in the following image:
+Starting with version 3.4 the HTTP resolver is available to retrieve user information from any kind
+of web service API. privacyIDEA issues a request to the target service and expects a JSON object in return.
+The configuration of the HTTP resolver sets the details of the request in the ``Request Mapping`` as well as the
+mapping of the obtained information as a ``Response Mapping``.
 
-.. figure:: images/http_resolver_1.png
+.. figure:: images/http_resolver.png
    :width: 500
 
-
-The important things here are ``Request Mapping`` and ``Response Mapping``, both of them using JSON format.
-
-``Request Mapping`` is used to inject the privacyidea's `userid` parameter into the correct place when you call to your HTTP API.
-
-Example 1:
-
-You have a HTTP endpoint with the following definition:
-
-   GET /users?id=<user_id>
-
-So you need to configure the request mapping like this:
-   
-   { "id": "{userid}" }
-
-Example 2:
-
-You have a HTTP endpoint with the following definition:
+The ``Request Mapping`` is used to build the request issued to the remote API from privacyIDEA's user information.
+For example an entpoint definition::
```

Comment:
instead of `::` you can use `.. sourcecode:: http` to get some syntax highlighting.